vnext/Shared/IDevSupportManager.h

```diff
@@ -21,6 +21,9 @@ struct IDevSupportManager {
       const uint16_t sourceBundlePort,
       std::function<void()> onChangeCallback) = 0;
   virtual void StopPollingLiveReload() = 0;
+
+  virtual void startInspector(const std::string &packagerHost, const uint16_t packagerPort) = 0;
```

Comment:
>startInspector [](http://example.com/codeflow?start=15&length=14)

Since we declare this interface in our code, could you please make sure that we use a consistent naming conventionsin side of the same type: previous methods in this interface were Pascal-ased and the new are camelCased.
Also, can we make these methods to be 'noexcept'?